tsconfig.json

```diff
@@ -0,0 +1,67 @@
+{
+    "compilerOptions": {
+      /* Basic Options */
+      "target": "ES5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+      "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+      // "lib": [],                             /* Specify library files to be included in the compilation. */
+      // "allowJs": true,                       /* Allow javascript files to be compiled. */
+      // "checkJs": true,                       /* Report errors in .js files. */
+      // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+      "declaration": true, /* Generates corresponding '.d.ts' file. */
```

Comment:
I don't think this option is being used

Reply:
I'm not 100% sure of the purpose of this option, but verified it does not affect the app. removed.